src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts\Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts\Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts\Avenir.ttc);
+  font-style: bold;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
```

Comment:
move it to the `html` and let inputs inherit it

Reply:
haven't fixed

Reply:
I can't move the box-sizing to the html tag. The font doesn't work both in the html tag and in the body tag.

Reply:
move to the `html` and add `font-family: inherit` to your inputs

Reply:
I added font-family: inherit. Thank you very much for your help!

Reply:
The test is positive))